### PR TITLE
Add commit_info.time

### PR DESCRIPTION
**Pull request by varac**

### CODE REVIEW

File: src/pytest_benchmark/utils.py [hunk 1]
```diff
@@ -186,13 +186,19 @@ def get_commit_info(project_name=None):
...
+            commit_time = check_output('git show -s --pretty=format:"%cI"'.split(),
+                                       universal_newlines=True).strip().strip('"')
         elif in_any_parent('.hg'):
             desc = check_output('hg id --id --debug'.split(), universal_newlines=True).strip()
             if desc[-1] == '+':
                 dirty = True
             commit = desc.strip('+')
+            commit_time = check_output('hg tip --template "{date|rfc3339date}"'.split(),
+                                       universal_newlines=True).strip().strip('"')
+            print(commit_time)
```
Comment (ionelmc) [inline src/pytest_benchmark/utils.py]: There's a leftover debug print here.

**Comment by varac** [comment]
ugh, sorry, updated.

**Comment by ionelmc** [comment]
Also, there should be a default for commit_time (that's why some tests fail).

File: src/pytest_benchmark/utils.py [hunk 2]
```diff
@@ -186,20 +186,26 @@ def get_commit_info(project_name=None):
...
             'id': commit,
+            'time': commit_time,
             'dirty': dirty,
             'project': project_name,
             'branch': branch,
         }
     except Exception as exc:
         return {
             'id': 'unknown',
+            'time': '1970-01-01T01:00:00+01:00',
```
Comment (ionelmc) [inline src/pytest_benchmark/utils.py]: I would put a `None` instead as an actual date would create a confusing situation where whoever reads the json can't really be sure if the commit was in the past, the commit has wrong date or date simply wasn't available.

Comment (varac) [inline src/pytest_benchmark/utils.py]: ok, will push this.

**Comment by varac** [comment]
I also added a default for commit_time now.

File: src/pytest_benchmark/utils.py [hunk 3]
```diff
@@ -186,20 +186,26 @@ def get_commit_info(project_name=None):
...
         elif in_any_parent('.hg'):
             desc = check_output('hg id --id --debug'.split(), universal_newlines=True).strip()
             if desc[-1] == '+':
                 dirty = True
             commit = desc.strip('+')
+            commit_time = check_output('hg tip --template "{date|rfc3339date}"'.split(),
+                                       universal_newlines=True).strip().strip('"')
         return {
             'id': commit,
+            'time': commit_time,
```
Comment (ionelmc) [inline src/pytest_benchmark/utils.py]: If there's no git/hg repo this would fail as commit_time is undefined.

**Comment by varac** [comment]
Danm, your're right. Fixed it.

**Comment by varac** [comment]
and rebased.

File: src/pytest_benchmark/utils.py (hunk 3 above)
Comment (varac) [inline src/pytest_benchmark/utils.py]: Danm, your're right. Fixed it.

**Comment by ionelmc** [comment]
I updated the changelog for you, let me know if attribution is wrong (or make pr to fix it).

**Pull request merged**